.github/workflows/clp-execution-image-build.yaml

```diff
@@ -3,55 +3,89 @@ name: "clp-execution-image-build"
 on:
   pull_request:
     paths:
+      - ".github/actions/clp-execution-image-build/action.yaml"
       - ".github/workflows/clp-execution-image-build.yaml"
-      - "components/core/tools/scripts/lib_install/*"
-      - "tools/docker-images/clp-execution-base-focal/**"
+      - "tools/docker-images/**"
```

Comment:
In my experience, `/**/*` is safer if we want to recursively get all the files and directories. `/**` can have [unexpected behaviour](https://github.com/y-scope/clp/pull/232) depending on the glob implementation.